DISCOVER/07_code_of_conduct.md

```diff
@@ -69,19 +69,23 @@ Make sure that **everyone involved in your conference/event** is aware that the
 **Why You Need a Code of Conduct**
 - [Why to have a CoC and how to make one](http://incisive.nu/2014/codes-of-conduct/)
 - [Why conferences need a code of conduct](https://jacobian.org/writing/codes-of-conduct/)
-- [You literally cannot pay me to speak without a Code of Conduct](http://rachelnabors.com/2015/09/01/code-of-conduct/)
+<!--NOT AVAILABLE- [You literally cannot pay me to speak without a Code of Conduct](http://rachelnabors.com/2015/09/01/code-of-conduct/)-->
```

Comment:
This one seems gone, probably need to remove it as there are many other examples.